pkgs/top-level/all-packages.nix

```diff
@@ -15555,7 +15555,7 @@ in
   libxmp = callPackage ../development/libraries/libxmp { };
 
   libxslt = callPackage ../development/libraries/libxslt {
-    python = if stdenv.isDarwin then python2 else python3;
+    python = python3;
```

Comment:
I think we should remove the overwrite entirely and just use python3 in inputs. Thats less confusing and more straight forward.

Reply:
There's quite a few packages in a similar boat. Does it make sense to open a separate PR to do a sweep of them? Or should I just do this one here?

Reply:
Hmm, `libxml2` has `python?isPy2` checks, and `libxslt` depends on python bits in `libxml2`. I don't think I can properly untangle this without breaking something along the way. Even renaming `python` to `python3` in inputs can break somebody's overlay.

Reply:
For this PR only this package should be changed. If you feel motivated to change the other packages you can do a treewide PR and if it has zero rebuilds we can merge. If it rebuilds things it should be done package by package or in smaller batches of maybe 5 to make it reviewable. 

I don't have a problem with breaking python 2 combability. We removed moved packages using it and upgraded them to python 3 and it is no longer build by default which means most packages are not tested on python 2 anymore.

Reply:
Fair enough, updated to make `libxslt` take `python3` as input.